spec.html

```diff
@@ -6235,6 +6236,63 @@
         1. If _result_ is an abrupt completion, perform HostReportErrors(&laquo; _result_.[[Value]] &raquo;).
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-agents">
+    <h1>Agents</h1>
+
+    <p> An <dfn id="agent">agent</dfn> comprises a set of ECMAScript execution contexts, an execution context stack, a running execution context, a set of named job queues, an <dfn id="agent-record">Agent Record</dfn>, and an <dfn id="executing-thread">executing thread</dfn>.  Except for the executing thread, the constituents of an agent belong exclusively to that agent. </p>
```

Comment:
Eh on second thought you can keep these in if you like :-P I try to be conservative with making new IDs since each new one is a maintenance burden but these are unlikely to be problems.